.gitmodules

```diff
@@ -1,21 +1,29 @@
-[submodule "isl"]
-       path = isl
-       url = git://repo.or.cz/isl.git
+[submodule "isl"] 
+        path = isl
+        url = git://repo.or.cz/isl.git
+        ignore = untracked
```

Comment:
Fix white space change above. You can use a "git diff --check" to check for whitespace errors.